app/controllers/HomeController.scala

```diff
@@ -61,6 +63,15 @@ class HomeController @Inject()(db: Database, cc: ControllerComponents, service:
     }
   }
 
+  def getMockQueue(patient: Patient): Future[Seq[(ClinicQueue, Option[Ticket], Doctor)]] = {
+    val queue = PublicQueue(1, DateTime.now().minusHours(1), DateTime.now(), Seq(1, 2, 3))
+    val ticket = Some(Ticket(1, new Patient(7, "John", "Doe")))
+    val doctor = new Doctor(6, "Doctor", "House")
+    Future {
```

Comment:
You can just Future(something) for one instruction expression

Reply:
👍